.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Use OCaml ${{ matrix.ocaml-version }}
-        uses: ocaml/setup-ocaml@v2
+        uses: ocaml/setup-ocaml@v2.0.0-beta3
```

Comment:
I'm a bit surprised that this is necessary, because the latest version of `v2` is supposed to be used by default, but perhaps there's a cache of the old version somewhere.  So, I think it's OK to use the explicit version, at least for now.